opds_import.py

```diff
@@ -1306,6 +1307,18 @@ def _detail_for_elementtree_entry(cls, parser, entry_tag, feed_url=None):
         if series_tag:
             data['series'], data['series_position'] = cls.extract_series(series_tag[0])
 
+        issued_tag = parser._xpath(entry_tag, 'dcterms:issued')
+        if issued_tag:
+            date_string = issued_tag[0].text
+            # By default, the date for strings that only have a year will
+            # be set to January 1 rather than the current date.
+            default = datetime.datetime(datetime.datetime.now().year, 1, 1)
+            try:
+                data["published"] = dateutil.parser.parse(date_string, default=default)
```

Comment:
I'm not sure if I should be using issued or published for this. It might not be consistent between OPDS feeds.

Reply:
I meant if I should store it as Edition.issued or Edition.published. published seems safer - in our feeds we put either issued or published as dc:issued depending on what we have.

Reply:
I agree published is safer.